yoyodyne/schedulers.py

```diff
@@ -64,6 +71,41 @@ def lr_lambda(self, step: int) -> float:
         return self.decay_factor * step**-0.5
 
 
+class LinearDecay(optim.lr_scheduler.LinearLR):
+    """Linear decay scheduler."""
+
+    optimizer: optim.Optimizer
```

Comment:
I think these type decls are no longer needed since the superclass can take care of it.